src/collectors/aerospike/aerospike.py

```diff
@@ -61,8 +63,15 @@ def get_default_config(self):
             'objects',
             'client_connections',
             'index-used-bytes-memory',
+            # 3.9 Stats
+            'objects',
```

Comment:
some things are listed twice, maybe group them under `# 2.7 and 3.9 Stats` or `# All version stats`